content/blog/blog-2020-07-10-webhook-development.md

```diff
@@ -0,0 +1,153 @@
+---
+date: 2020-07-10
+---
+
+# Developing and Testing Kubernetes Webhooks
+
+Kubernetes development, testing and debugging can be challenging by itself, but fewer things are more challenging than debugging webhooks or technologies like them.  Specifically components which require a callback AND that callback requires a valid certificate to succeeded.  See requests for [Come up with webhook developer workflow to test it locally](https://github.com/kubernetes-sigs/kubebuilder/issues/400), I thought it would be valuable to the community to post a solution.  
```

Comment:
```suggestion
Kubernetes development, testing and debugging can be challenging by itself, but fewer things are more challenging than debugging webhooks or technologies like them.  Specifically components which require a callback when that callback requires a valid certificate to succeed.  Seeing requests for [Come up with webhook developer workflow to test it locally](https://github.com/kubernetes-sigs/kubebuilder/issues/400), I thought it would be valuable to the community to post a solution.  

<!-- more -->
```